openmc/cppapi/__init__.py

```diff
@@ -0,0 +1 @@
+from .openmc_cpp import *
```

Comment:
Ok, so I think as far as naming, I'd like this to be called `core` on the Python side, i.e.
```Python
import openmc.core

openmc.core.init()
...
```
So, this file can be moved to `openmc/code/__init__.py`. I think the actual name of the shared library `openmc_cpp...` is fine.